src/rendermime/rendermime.ts

```diff
@@ -146,7 +146,14 @@ class RenderMime {
       if (model.data.has(mimeType)) {
         let options = { mimeType, model, sanitizer };
         let renderer = this._renderers[mimeType];
-        if (renderer.canRender(options)) {
+        let can_render = false;
+        try {
+          can_render = renderer.canRender(options);
+        } catch (err) {
+          console.error("Got an error when checking the renderer for the mimeType '"
```

Comment:
Will use the format string 👍 

The newline is for better readability. If the newline is not there, it prints the exception right after the string we wrote:
```javascript
>>> console.log("abcde ", SyntaxError("this is a syntax error"))
abcde  SyntaxError: this is a syntax error
    at <anonymous>:1:23
```

If the newline is there, it gets formatted nicely to:
```javascript
console.log("abcde \n", SyntaxError("this is a syntax error"))
abcde 
 SyntaxError: this is a syntax error
    at <anonymous>:1:25
```